sharktank/sharktank/evaluate/perplexity_iree.py

```diff
@@ -387,7 +365,9 @@ def run_iree_module(devices: list[iree.runtime.HalDevice]):
                 if skip_decode or len(out_logits) == 0:
                     token_batch = self.token_ids[:, : i + 1]
 
+                    start = time.time()
```

Comment:
Accuracy issues in time.time() will likely start being an issue here for our smaller models. I think it would be best to replace this with time.time_ns() instead and do the conversion from that.